pkg/controller/cloud-controller-cap/capability.go

```diff
@@ -0,0 +1,120 @@
+package cloudcontrollercap
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	configv1 "github.com/openshift/api/config/v1"
+	openshiftconfigclientset "github.com/openshift/client-go/config/clientset/versioned"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/apimachinery/pkg/util/yaml"
+	kscheme "k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/rest"
+	"k8s.io/klog/v2"
+)
+
+const installConfigFilename = "cluster-config.yaml"
+
+var (
+	ManifestsDir    = "/etc/mcc/bootstrap"
```

Comment:
this is an interesting approach. When I was talking about storing it in the controller I meant storing it literally in one of the `type Controller struct` objects. This could be causing weird things to happen.